src/core/include/mp-units/framework/representation_concepts.h

```diff
@@ -309,15 +310,45 @@ concept VectorRepresentation = (!is_quantity<T>) && Vector<T> && requires(const
 
 MP_UNITS_EXPORT template<typename T>
 concept Representation = detail::ScalarRepresentation<T> || detail::ComplexRepresentation<T> ||
-                         detail::VectorRepresentation<T>;  // || detail::TensorRepresentation<T>;
+                         detail::VectorRepresentation<T>;  // || detail::TensorRepresentation<T>; */
 
 namespace detail {
 
+#ifdef MP_UNITS_APPLE_CLANG_HACKS
+template<typename T>
+constexpr bool is_weakly_regular = std::copyable<T> && std::equality_comparable<T>;
```

Comment:
It might, in general the fix for the crash seemed to be to reduce the number of concepts being used. I can try to slowly add back in concepts now that it is fast and not crashing and see what happens.